cnf/Makegap.in

```diff
@@ -47,6 +47,7 @@ gapdll: $(OBJECTS) $(ITANIUMOBJ) $(EXTOBJS)
 
 # dependencies are automatically generated
 ariths.o: ../../Makefile @srcdir@/ariths.c @srcdir@/system.h config.h @srcdir@/gasman.h \
+<<<<<<< HEAD
```

Comment:
Woops, although Makegap.in isn't actually used in HPC-GAP, so I might just delete it to get it out of the way.

Chris




On 29/09/2015, 22:23, "Alexander Konovalov" <notifications@github.com> wrote:

>In cnf/Makegap.in <https://github.com/gap-system/gap/pull/267#discussion_r40731801>:
>> @@ -47,6 +47,7 @@ gapdll: $(OBJECTS) $(ITANIUMOBJ) $(EXTOBJS)
>>  
>>  # dependencies are automatically generated
>>  ariths.o: ../../Makefile @srcdir@/ariths.c @srcdir@/system.h config.h @srcdir@/gasman.h \
>> +<<<<<<< HEAD
><<<<<< is a relic from resolving diffs. 
>—
>Reply to this email directly or 
>view it on GitHub <https://github.com/gap-system/gap/pull/267/files#r40731801>.
>
>